src/qt/coincontroldialog.cpp

```diff
@@ -2,10 +2,15 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#if defined(HAVE_CONFIG_H)
+#include "config/bitcoin-config.h"
+#endif
+
 #include "coincontroldialog.h"
 #include "ui_coincontroldialog.h"
 
 #include "addresstablemodel.h"
+#include "base58.h"
```

Comment:
These changes seem out of place here...?

Reply:
These extra includes are necessary now that `paymentserver.h` doesn't include them anymore.